### DISCUSSION TITLE
BFE配置后端服务实例是否仅支持IP形式

### DISCUSSION THREAD

**Discussion by ZhongShawn** [bfenetworks/bfe#1118] [Q&A (问题和回答)] [answered]
我这边的后端服务是一个域名，考虑到指定IP的话，当解析的IP地址有变化时，将无法访问。因此想配置域名方式的后端实例。BFE是否不支持域名的形式？如果不支持的话，有没有什么办法能解决？

**Comment by youerning** [comment, marked as answer]
我测试v1.6.0版本是没有问题，看源代码也没有阻止不能使用域名的判断。

你可以这样设置后端`"Addr": "example.com"`
